src/main/java/core/basesyntax/App.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operations.BalanceOperation;
+import core.basesyntax.strategy.operations.PurchaseOperation;
+import core.basesyntax.strategy.operations.ReturnOperation;
+import core.basesyntax.strategy.operations.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class App {
+
+    private static final String INPUT_DATA_PATH
+            = "/home/aos/IdeaProjects/jv-fruit-shop/src/main/java/resources/fileFrom.csv";
+    private static final String OUTPUT_DATA_PATH
+            = "/home/aos/IdeaProjects/jv-fruit-shop/src/main/java/resources/report.csv";
```

Comment:
```suggestion
            = "src/main/java/resources/report.csv";
```
The same. an absolute path